backend/app/assets/stylesheets/spree/backend/globals/mixins/_media.scss

```diff
@@ -0,0 +1,28 @@
+/// Outputs a media-query block.
+///
+/// @param {String} $media-query
+///   A string corresponding to the media query to target.
+///
+/// @example scss - Usage
+///   .responsive-element {
+///      color: blue;
+///      @include media("screen and (min-width: 480px)") {
+///        color: red;
+///      }
+///   }
+///
+/// @example css - CSS Output
+///  .responsive-element {
+///    color: blue;
+///  }
+///  @media screen and (min-width: 480px) {
+///    .responsive-element {
+///      color: red;
+///    }
+///  }
+
+@mixin media($media-query) {
```

Comment:
@graygilmore thanks for your feedback! We thought that having a mixin would be easier to add features to media query in future development, without updating code in the whole project. Also the mixin version seems to be a little more user friendly and more "readable". 

But your suggestion also makes sense and we're open to change code in this direction if you want.